includes/abstracts/abstract-ur-form-field.php

```diff
@@ -140,13 +140,14 @@ public function frontend_includes( $data = array(), $form_id, $field_type, $fiel
 		if ( 'select' === $field_key ) {
 			$option_data = isset( $data['advance_setting']->options ) ? explode( ',', $data['advance_setting']->options ) : array(); // Backward compatibility. Modified since 1.5.7
 			$option_data = isset( $data['general_setting']->options ) ? $data['general_setting']->options : $option_data;
-
 			$options = array();
+
 			if ( is_array( $option_data ) ) {
 				foreach ( $option_data as $index_data => $option ) {
-					$options[ $option ]   = $option;
-					$form_data['options'] = $options;
+					$options[ $option ]   = ur_string_translation( $form_id, 'user_registration_' . $data['general_setting']->field_name . '_' . ( ++$index_data ), $option );
```

Comment:
@coderkoala Translation Field name should be like 'user_registration_field_name_option_1' .